jobs/rocoto/run_driver.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
```

Comment:
for ops, we need this script to call a J-job in jobs, and then that script needs to call an ex-script in scripts/ which can be a python script

Reply:
There is a j-job and an ex-script in obs-monitor.  They can be modified as needed. They are in /jobs and /scripts directories respectively.  Also there's a driver/runObsMon.sh, a test script which submits the j-job.